Makefile

```diff
@@ -116,3 +116,16 @@ commitlint:
 	@echo
```

Comment:
I renamed the PR to "chore", because it's not really a "feature" for kuberpult operators.